src/lib/analytics.js

```diff
@@ -85,6 +86,12 @@ function postAnalytics(data) {
       }
 
       data.ci = isCI();
+
+      data.environment = {};
+      if (isStandalone) {
```

Comment:
```suggestion
      if (!isStandalone) {
```

We should check the opposite - in standalone mode we ship (and control) node/npm version. We care about the non-standalone ones.

Check whether it affects logic in any test as well 🤞